html/pfappserver/lib/pfappserver/Base/Form/Role/WithSource.pm

```diff
@@ -14,17 +14,22 @@ use HTML::FormHandler::Moose::Role;
 with 'pfappserver::Base::Form::Role::Help';
 
 use pf::log;
+use pf::constants;
 
 after 'setup' => sub {
     my ($self) = @_;
 
     if($self->for_module->does('captiveportal::Role::MultiSource')){
-        $self->field('source_id')->multiple(1);
-        $self->field('source_id')->options([$self->options_sources(multiple => 1)]);
+        $self->field('multi_source_ids.contains')->options([$self->options_sources(multiple => 1)]);
+        $self->field('multi_source_ids')->inactive($FALSE);
```

Comment:
Tell that to the formhandler team :P 
http://search.cpan.org/dist/HTML-FormHandler-0.40017/lib/HTML/FormHandler/Field.pm#Field_data